soc/nordic/nrf54h/ironside_se/Kconfig

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2025 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_NRF54H20_IRONSIDE_SE_BOOT_REPORT
+	bool "Nordic IRONside SE boot report"
+	default y if SOC_NRF54H20_CPUAPP
+	depends on SOC_NRF54H20_IRON
```

Comment:
Right now we depend on building for cpuapp as well - I guess there is no other core that sets the IRON config until cpurad gets added, but should we add a depends on regardless for correctness? Better to see it fail at build time than run time if/when cpurad gets added

Reply:
We use this for CPSEC, I will see if there is some other way to get around that.